docs/labwc-config.5.scd

```diff
@@ -433,6 +433,43 @@ windows using the mouse.
 	<mouse><mousebind> entries exist, the same default mousebinds will be
 	loaded even if the <default /> element is not provided.
 
+## TABLET
+
+```
+<tablet rotate="0">
+  <area top="0.0" left="0.0" width="0.0" height="0.0">
+  <map button="tip" to="left" />
+  <map button="stylus" to="right" />
+  <map button="stylus2" to="middle" />
+</tablet>
+```
+
+*<tablet><area top="mm" left="mm" width="mm" height="mm" />*
+	The active tablet area can be specified by setting the *top*/*left*
+	coordinate (in mm) and/or *width*/*height* (in mm). If width or
+	height are omitted or default (0.0), width/height will be set to
+	the remaining width/height seen from top/left.
```

Comment:
Can we start this with a short explanation on why this setting may be useful for the user and that its completely optional?

Reply:
Done